src/walletConnect/screens/useIsDappListed.ts

```diff
@@ -1,16 +1,19 @@
 import { useSelector } from 'react-redux'
-import { dappsListSelector } from 'src/dapps/selectors'
+import { dappConnectInfoSelector, dappsListSelector } from 'src/dapps/selectors'
+import { DappConnectInfo } from 'src/dapps/types'
 
 export const useIsDappListed = (dappName?: string, dappUrl?: string) => {
   const dappsList = useSelector(dappsListSelector)
+  const dappConnectInfo = useSelector(dappConnectInfoSelector)
 
   return (
-    !!dappName &&
-    !!dappUrl &&
-    !!dappsList.find(
-      (dapp) =>
-        dapp.name.toLowerCase() === dappName.toLowerCase() &&
-        new URL(dapp.dappUrl).origin === new URL(dappUrl).origin
-    )
+    dappConnectInfo === DappConnectInfo.None ||
```

Comment:
I still think this shouldn't be here. The fact that the dapp is listed or not doesn't depend on the value of this remote config.

If this hook was named `useShouldShowLogo` or something, that would be fine. Would be still interesting to have a separate `useIsDappListed` though.